resources/assets/js/app.js

```diff
@@ -4,12 +4,9 @@ const input = document.getElementById('url');
 form.addEventListener('submit', event => {
```

Comment:
Is this client side validation still needed? I would prefer checking everything serverside, since you still can browse to `https://dnsrecords.io/SPATIE.BE`. Btw, seems to work without this script allready .